biolink-model.yaml

```diff
@@ -4853,7 +4904,6 @@ classes:
       - DOID
       - OMIM
       - ORPHANET
-      # - ORPHA  # no idea what this prefix resolves to globally. Is is an abbrevation for ORPHANET? not used in model
```

Comment:
In Monarch the orphanet prefix has had three iterations - as a case study for transient prefixes:

https://github.com/monarch-initiative/dipper/issues/73
https://github.com/monarch-initiative/dipper/issues/469

Reply:
I went through those tickets and I still am not sure what the canonical prefix is.

Monarch is on using `ORPHA` but identifiers.org is still on `ORPHANET`.